packages/Select/src/Select.js

```diff
@@ -1,94 +1,139 @@
 import * as constants from "@paprika/constants/lib/Constants";
+import SelectPropsCollector from "./SelectPropsCollector";
 import * as sc from "./Select.styles";
 
 const Select = React.forwardRef((props, ref) => {
-  const renderPlaceholder = () => {
-    const { placeholder } = props;
-    if (!placeholder) return null;
-    return (
-      <option disabled value="" key="placeholder">
-        {placeholder}
-      </option>
-    );
-  };
-
   const {
     a11yText,
-    className,
     children,
+    defaultValue,
     hasError,
     isDisabled,
     isReadOnly,
-    size,
+    onChange,
     placeholder,
+    size,
     value,
     ...moreProps
   } = props;
 
-  if (a11yText || placeholder) {
-    moreProps["aria-label"] = a11yText || placeholder;
+  const isControlled = value !== undefined;
+  const containerProps = extractChildrenProps(children, SelectPropsCollector);
+  const [hasNoValue, setHasNoValue] = React.useState(isControlled ? !value : !defaultValue);
+
+  const _refSelect = React.useRef();
+  const refSelect = ref || _refSelect;
+
+  const styleProps = {
+    hasError,
+    isDisabled,
+    isPlaceHolderSelected: !!placeholder && hasNoValue,
+    isReadOnly,
+    size,
+  };
+
+  function handleChange(event) {
+    const newValue = event.target.value;
+    if (hasNoValue && newValue !== "") {
+      setHasNoValue(false);
+    }
+    if (!hasNoValue && newValue === "") {
+      setHasNoValue(true);
+    }
   }
 
-  const rootClasses = classNames(
-    "form-select",
-    `form-select--${size}`,
-    { "form-select--placeholder": !value && placeholder },
-    { "form-select--is-disabled": isDisabled },
-    { "form-select--is-readonly": isReadOnly },
-    { "form-select--has-error": hasError },
-    className
-  );
+  const renderPlaceholder = () => {
+    if (!placeholder) return null;
+    return (
+      <option disabled value="" data-pka-anchor="select.placeholder">
+        {placeholder}
+      </option>
+    );
+  };
+
+  function getDefaultValue() {
+    if (isControlled) return undefined;
+    if (defaultValue === null && !!placeholder) return "";
+    return defaultValue;
+  }
 
   return (
-    <sc.Select className={rootClasses} data-pka-anchor="select">
-      <select
+    <sc.SelectContainer data-pka-anchor="select.container" {...containerProps}>
+      <sc.Select
         aria-invalid={hasError}
-        className="form-select__select"
+        aria-readonly={isReadOnly}
+        aria-label={a11yText}
+        data-pka-anchor="select"
+        defaultValue={getDefaultValue()}
         disabled={isDisabled || isReadOnly}
-        ref={ref}
+        onChange={callAll(handleChange, onChange)}
+        ref={refSelect}
         value={value}
+        {...styleProps}
         {...moreProps}
       >
         {renderPlaceholder()}
         {children}
-      </select>
-    </sc.Select>
+      </sc.Select>
+      <sc.CaretIcon data-pka-anchor="select.arrow-icon" isDisabled={isDisabled || isReadOnly} />
```

Comment:
Instead of embedding the SVG image in the background, I've decided to make this consistent with the `<ListBox>` and use the icon component overlayed on top of the `<select>` element.